legacy/application/controllers/ShowbuilderController.php

```diff
@@ -234,23 +234,23 @@ public function scheduleAddAction()
         $mediaItems = $request->getParam('mediaIds', []);
         $scheduledItems = $request->getParam('schedIds', []);
 
-        $log_vars = [];
-        $log_vars['url'] = $_SERVER['HTTP_HOST'];
-        $log_vars['action'] = 'showbuilder/schedule-add';
-        $log_vars['params'] = [];
-        $log_vars['params']['media_items'] = $mediaItems;
-        $log_vars['params']['scheduled_items'] = $scheduledItems;
-        Logging::info($log_vars);
+        // $log_vars = [];
```

Comment:
Why not remove this instead of commenting it out?

Reply:
I wanted to keep them for development, so I should probably not comment them, but only change the logging level to debug.